### PR TITLE
Added Script to generate valid google search url with required parame…

### LINKED ISSUE #461: Fetch common tags of top google searches
# Description

This Script will fetch the common tags of top google searches for SEO optimization and indexing.

## Language

- [x] Python

## Type of issue

- [x] Feature

## Checklist:

- **For Feature**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Fetching list of top used tags | This script will fetch the list of tags used by top google searches will find the common tags in them for using for SEO | Python, SLoP, Hacktoberfest | @GoswamiTushar |

**Is your feature request related to a problem? Please describe.**: 
While having a conversation with my friend who's leaning SEO and SEP, he told me regarding the role of tags in html and SEO.
The problem is manually searching for tags is boring and time consuming. This script will automate the task of extracting tags which are used by initial google searches and make a list of them for user to use them in their website.

**Describe the solution you'd like**: 
A simple python list will automate the scraping task along with some additional functionality of visualizing the frequently used tags.

### PR DESCRIPTION
**Pull request by GoswamiTushar**
# Description

URL Generator is an independent script with scope of adding more functionality for other search engines as well. 
It takes the search query as input and returns the well formatted URL string.

Fixes #461

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by HarshCasper** [comment]
@seema1711 I would like if you can go over these Scripts because I feel that I might be missing a lot of details in here, in an otherwise brilliant execution

File: Python/Tags_Scraper/Requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Libraries which will be required and need to be installed are:
```
Comment (HarshCasper) [inline Python/Tags_Scraper/Requirements.txt:1]: You don't need to add this line in your `requirements.txt`

You can generate a Requirements File via `pip freeze` and remove all the unnecessary lines.

File: Python/Tags_Scraper/Readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+This project is modularized in 4 Modules
+
+- **URL_generator** : It's an indipendent module which generates Google accepted URL with all the valid paramenters. It can be extended further by adding support for other Search engines as well.
+
+- **SearchResults** : SearchResults module takes the search query as input and returns the list of links of div tag, which are searched result's pages link.
+
+- **Fetch_Tags** : Fetch_Tags module takes in the list of URLs generated by SearchResults , visits each URL, scrapes data of Title tag, H2 and H3 tag and alt tag and count for how many times each word has been used in different webpages.
+
+- **Main** : Main module is what we need to interact with. It combines the functionality of other three modules into 1 whole. The data generated is Stored in KeywordsData.txt file.
```
Comment (HarshCasper) [inline Python/Tags_Scraper/Readme.md:15]: Can you add a brief note on how to run your Scripts?

File: Python/Tags_Scraper/Fetch_Tags.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		try :
+			response = requests.get(url, headers = self.usr_agent, verify = False)
+			response.raise_for_status()
+			return response.text
+		except:
+			return None
+
+	def get_keywords(self, html):
+		soup = BeautifulSoup(html, 'html.parser')
+		# print(soup.prettify())
```
Comment (HarshCasper) [inline Python/Tags_Scraper/Fetch_Tags.py:46]: You can remove the Commented Code.

File: Python/Tags_Scraper/Fetch_Tags.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+			for line in h3_list:
+				if line != None:
+					for word in line.split():
+						if word not in special_characteres:
+							self.h3_dict[word] = self.h3_dict.get(word, 0) + 1
+
+		# print(self.alt_dict, self.title_dict, self.h2_dict, self.h3_dict, sep = '\n')
+		
+
+
```
Comment (HarshCasper) [inline Python/Tags_Scraper/Fetch_Tags.py:123]: Remove the Blank Lines of Code

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Can you try running your Code through PyLint and improving on the pointers given by the Tool? Also kindly add a Screenshot/GIF in your README to help better understand on how your Script works.

**Comment by HarshCasper** [comment]
@GoswamiTushar Are you working on this? Kindly let me know so that I can award you the points for the same aptly.

**Comment by GoswamiTushar** [comment]
> @GoswamiTushar Are you working on this? Kindly let me know so that I can award you the points for the same aptly.

Yes I'm working on it. 
I'm using pylint for the first time so its taking a bit time to optimise code in a way that it gets score over 8.
I'll get it done by 29 oct

**Pull request closed without merging**